bin/percona-qan-agent/main.go

```diff
@@ -304,6 +306,11 @@ func run(agentConfig *pc.Agent) error {
 		},
 	)
 
+	// Start http server for pprof
+	go func() {
+		golog.Println(http.ListenAndServe("localhost:6060", nil))
+	}()
+
```

Comment:
@AlekSi @cfsalguero @delgod Is it safe to add pprof at `localhost:6060` to `qan-agent`? Or I shouldn't do that for some reason?

Reply:
I would make three tweaks:
1. use `127.0.0.1` instead of `localhost` to prevent "interesting" IPv6 and dual-stack related issues;
2. enable it with with a flag, make it disabled by default; maybe that flag should set s listen address to solve the first problem too;
3. replace `Println` with `Fatal` – if debug server was explicitly requested with a flag, we should fail if we can't start it.

Reply:
I just dropped it for now, it's already to many changes in this PR, we can add it later with separate PR.